### ISSUE TITLE
Restoring multiple autosaved fields on one page

### ISSUE THREAD

**Issue by bemoody** [w8tcha/CKEditor-AutoSave-Plugin#133]
If there are multiple CKEditor input fields on a single page, each field is auto-saved separately (which is good!)

However, if I make changes in several fields, then close the window and re-open it, I am presented with the same question several times in a row ("Would you like to compare content versions and choose which one to load?")

This isn't really helpful; better would be to ask this initial question only once per page.

Still better would be to show the diffs for *all editors on the page* in a single dialog.

**Comment by w8tcha** [comment]
Yes i understand the problem. This will take some time until if find a solution.
